src/components/AboutContent.vue

```diff
@@ -6,10 +6,8 @@
 
         <h3>Questions and comments</h3>
         <p>
-          The library is intended to be lightweight and focused purely in
-          dealing with CellML models. In achieving this, some of the convenience
-          methods/objects that we might expect will not be part of libCellML.
-          For questions, feature requests, and bug reports about the libCellML
+          The library is designed to be used for working with CellML models. For
+          questions, feature requests, and bug reports about the libCellML
```

Comment:
Minor: we really ought to have one sentence per line. Makes it much easier to review changes.

Reply:
Can't it is decided by eslint.

Reply:
Ok, annoying, but fair enough.